tools/model_tools/src/openvino/model_zoo/open_model_zoo.py

```diff
@@ -0,0 +1,193 @@
+import cv2
+import os
+import argparse
+import yaml
+
+from pathlib import Path
+
+from  openvino.model_zoo import (
+    _configuration, _common, omz_downloader, omz_converter
+)
+from openvino.model_zoo.download_engine import validation
+
+
+class Model:
+    def __init__(self, name=None, model_path=None, description=None, task_type=None, subdirectory=None):
+        self.model_path = model_path
+        self.description = description
+        self.task_type = task_type
+        self.name = name
+        self.subdirectory = subdirectory
+
+        self.net = None
+        self.exec_net = None
+        self._accuracy_config = None
+        self._model_config = None
+        self._model = None
```

Comment:
is it possible to provide ie into consturtor and create network, when it will be necessary as alternative of providing it into input/output getters and inference?